### DISCUSSION TITLE
Stopped working after W11 update (Dual Shock 3)

### DISCUSSION THREAD

**Discussion by dhstsw** [nefarius/DsHidMini#275] [Support]
Hi there,
As per subject, i can't manage to make it work again.
Initially (after update), only wired was working.
Then i reinstalled BthPS3, as per DsHidMini config tool it was paired (again) but not seen on DS4Windows.
Then went through all the docs for troubleshooting in DS4Windows but nothing.

Situation ATM is as follows:

1) DsHidMini Control sees the controller, both wired and BT (paired);
2) DS4Windows sees the controller, bothh wired and BT;
3) Controller is seen under devices as "PLAYSTATION(R)3 Controller"
![image](https://user-images.githubusercontent.com/59790194/212477477-aee0c7b9-2757-432c-a63a-59d1578a2932.png)
4) Gaming peripheals under windows DOESN'T see the controller:
![image](https://user-images.githubusercontent.com/59790194/212477509-ea4a7d29-e600-4874-9b3b-76c8a20f2a43.png)

I noticed, since upgrade, that if i do any modification with DsHidMini Control (eg: HID Device Mode) and then apply, it always fails to restart the controller (it was working fine before). I tried with other controllers (i have 3 DS3 and 3 SixAxis) and with all of them is the same situation.

Any hint/Help?

(of course i altready tried uninstall everything and start from zero).

**Comment by nefarius** [comment]
The gaming peripherals dialog not seeing the DS3 is normal and intended when in DS4Windows mode.

**Comment by Kanuan** [comment]
If I understood this correctly, no controller is shown in Windows' gaming peripherals even when DS3 is in DS4Windows mode while DS4Windows is running and is detecting the controller?

**Reply by dhstsw** [reply]
> If I understood this correctly, no controller is shown in Windows' gaming peripherals even when DS3 is in DS4Windows mode while DS4Windows is running and is detecting the controller?

Exactly :(

**Reply by Kanuan** [reply]
Export DS4Windows log while the controller is connected and then send the file here.

Please, do not directly copy and paste the text itself

**Reply by dhstsw** [reply]
> Export DS4Windows log while the controller is connected and then send the file here.
> 
> Please, do not directly copy and paste the text itself

Here it is.

[ds4windows_log.zip](https://github.com/ViGEm/DsHidMini/files/10418077/ds4windows_log.zip)

**Reply by dhstsw** [reply]
After some other fiddling now i have this:

![image](https://user-images.githubusercontent.com/59790194/212482268-61312b43-5af4-4410-9014-e26df753e98b.png)

-That controller name (Corì whatever) doesn't make any sense :-( But if i go on it's properties it beheaves as it should. Only that IT'S not seen by/forwaded to system/games.
EG: Gamepad Tester stays waiting for a controller to show up:
![image](https://user-images.githubusercontent.com/59790194/212482384-a9d4cf9a-c914-408e-9169-664ef0116ed0.png)

Also, it's being shown with that name only if i set the controller as DS4Windows in DsHidMini Control HID Mode. If i set it in any other mode (eg: Xinput) the name is correct.

Before that, i reinstalled everything. :-/

**Reply by dhstsw** [reply]
Uninstalled and reinstalled the ViGEm Bus Driver.
Now everything works again, still, the weird controller name is still there :-/
Well, if it works i guess i can live with it.
:)
